style.css

```diff
@@ -0,0 +1,391 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: 'Montserrat', sans-serif;
+}
+
+section {
+	background-color: #c0b1e1;
+	padding: 20px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.sw-logo {
+	font-family: 'Gloria Hallelujah', cursive;
+	font-size: 2.5rem;
+	text-indent: 20px;
+}
+
+.nav-bar {
+	word-spacing: 0.8rem;
+	padding: 20px 10px 10px 5px;
+}
+
+.nav-bar a[href*="about-me.html"] {
+	word-spacing: 0.1rem;
+}
+
+.nav-bar a {
+	text-decoration: none;
+	color: #000000;
+}
+
+.top-bar .hamburger {
+	display: none;
+}
+
+main {
+	background-color:#474c58;
+	color: #ffffff;
+	padding: 5%;
+}
+
+.homepage {
+	width: 100%;
+}
+
+.titlepage {
+	margin: 5%;
+	font-size: 1.75rem;
+	position: relative;
+}
+
+.title-format {
+	display: flex;
+	margin: 25px;
+}
+
+.title-letter {
+	border: 4px solid #ffffff;
+	padding: 17px;
+	margin: 20px;
+	height: 75px;
+	width: 75px;
+	text-align: center;
+	color: #000000;
+}
+
+#title-letter-1 {
+	background-color: #736a87;
+}
+
+#title-letter-1:hover {
+	background-color: #e6f0f5;
+}
+
+#title-letter-2 {
+	background-color: #9a8eb4;
+}
+
+#title-letter-2:hover {
+	background-color: #cee1ea;
+}
+
+#title-letter-3 {
+	background-color: #c0b1e1;
+}
+
+#title-letter-3:hover {
+	background-color: #b5d2e0;
+}
+
+#title-letter-4 {
+	background-color: #cdc1e7;
+}
+
+#title-letter-4:hover {
+	background-color: #8aa4b0;
+}
+
+#title-letter-5 {
+	background-color: #d9d0ed;
+}
+
+#title-letter-5:hover {
+	background-color: #79909a;
+}
+
+#title-letter-6 {
+	background-color: #e6e0f3;
+}
+
+#title-letter-6:hover {
+	background-color: #57676e;
+}
+
+#title-letter-7 {
+	background-color: #6a7e87;
+}
+
+#title-letter-7:hover {
+	background-color: #e0e1f3;
+}
+
+#title-letter-8 {
+	background-color: #8ea8b4;
+}
+
+#title-letter-8:hover {
+	background-color: #b1b5e1;
+}
+
+#title-letter-9 {
+	background-color: #b1d2e1;
+}
+
+#title-letter-9:hover {
+	background-color: #8e91b4;
+}
+
+#title-letter-10 {
+	background-color: #c8e0ea;
+}
+
+#title-letter-10:hover {
+	background-color: #7c7f9e;
+}
+
+.video-col video {
+	position: absolute;
+	right: 15%;
+	top: 30%;
+	/*width: 400px;
+	height: 350px;*/
```

Comment:
If you've commented these out, then presumably you don't them. So you might as well delete them.